onboarding-site/index.html

```diff
@@ -229,6 +234,12 @@ <h3>Next step: create your own Artifact and deploy it to your device</h3>
       let inventory = document.querySelector('#device-inventory');
       Object.entries(mender_inventory).map(([name, value]) => inventory.appendChild(mapToAttributeContainer(name, value, 'inventory-item')));
       
+      const backLink = document.querySelector('#next-steps a');
+      backLink.addEventListener('click', evt => {
+        evt.preventDefault();
+        const source = decodeURIComponent(window.location.search.substring(window.location.search.indexOf('?') + 1));
+        window.open(source ? source : 'https://hosted.mender.io/ui/#/releases');
```

Comment:
I'm very far from an expert in this area, so please ignore me if I'm completely off the mark, but isn't there a more "standard" way of dealing with URL parameters? Those that deal with `URL?param=value`, I mean.

Reply:
yes, there are URLSearchParams, relying on window.location.search - unfortunately the latter doesn't work as reliable as the method above - mostly due to the `#` that might come with the URL (+ we only anticipate a single param)

Then again for the on device site, this should not be a problem... I'll adjust the gui and here

Reply:
Thanks. What I'm a bit worried about is if we need to add more params later. Since we only host one single artifact, it needs to keep working with old versions of the onboarding, so it might be more compatible to do it now.

Reply:
should be ok now, but relies on https://github.com/mendersoftware/gui/pull/483 being merged as well